TS00-Brief-Tabular-Time-Series-In-R.Rmd

```diff
@@ -75,31 +69,22 @@ These data are proxy data for what will be available for 30 years
 on the [NEON data portal](http://data.neoninc.org/ "NEON data")
 for both Harvard Forest and other field sites located across the United States.
 
-####Setting the Working Directory
-The code in this lesson assumes that you have set your working directory to the
-location of the unzipped file of data downloaded above.  If you would like a
-refresher on setting the working directory, please view the
-[Setting A Working Directory In R]({{site.baseurl}}/R/Set-Working-Directory "R Working Directory Lesson")
+**Set Working Directory:** This lessons assumes that you have set your working 
+directory to the location of the downloaded and unzipped data subset. [An overview
+of setting the working directory in `R` can be found here.]({{site.baseurl}}/R/Set-Working-Directory "R Working Directory Lesson") 
 lesson prior to beginning this lesson.
```

Comment:
Raster Lesson 05 (maybe 06) has two include file buttons.  I didn't think it looked too weird when I built them locally.  We can look at it when discussing the grey box.  I'll add it to the issue in template repo where I'm pulling all the info together from all the repos.